src/components/DropDown.tsx

```diff
@@ -0,0 +1,17 @@
+import '../styles/DropDown.scss';
+
+const DropDown = ({ callback }: { callback: Function }) => {
+    return (
+        <div className="dropdown">
+            <button><span>Sort by </span><img src="/images/filter.svg" alt="icon filter" /></button>
+            <div className="dropdown-content">
+                <ul>
+                    <li value="0" onClick={(e) => callback(e)}>from A to Z</li>
```

Comment:
It is better to have enum, it will be easier not to forget which value stands for ascending and which one for descending sorting.